app/views/layouts/_topbar.html.haml

```diff
@@ -3,12 +3,12 @@
     %li.name
 
       %h1#Surveys
-        %a.home{:href => root_path} Surveys
+        %a.home{:href => root_path} Tech Collective
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace